ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -55,7 +55,7 @@ Ansible can run on almost anything, but in production scenarios, Ansible is typi
 Arista and accepts third-party contributions on GitHub at [aristanetworks/ansible-avd](https://github.com/aristanetworks/ansible-avd).
 
 While Ansible is the core automation engine, AVD is an Ansible Collection described above. It provides roles, modules, and plugins
-that allows the user to generate and deploy best-practice configurations to a layer three leaf-spine network.
+that allow the user to generate and deploy best-practice configurations to a layer three leaf-spine network.
```

Comment:
AVD supports more than l3ls now, so maybe the end of this sentence could be changed to "... configurations to Arista based networks of various different designs."

Reply:
addressed in 6e84056